src/js/components/Video/Video.js

```diff
@@ -48,7 +48,7 @@ const Video = forwardRef(
       alignSelf,
       autoPlay,
       children,
-      controls = 'over',
+      controls,
```

Comment:
I think we should do `controls: controlsProp` here and then use `[controls, setControls]` for the state below. "newControls" is a bit too time sensitive to the where we are today. In 9 months, it will seem odd.